### DISCUSSION TITLE
New AI image editing feature in Microsoft Edge browser version 121 (Canary):  Smart object erasing

### DISCUSSION THREAD

**Discussion by HotCakeX** [SpyNetGirl/MSEdgeFeatures#16] [Announcements]
Microsoft Edge version 121 (Canary) has a new hidden AI assisted photo editing feature.  It's called Smart Erase and it's only one of the many AI assisted features that are coming to the Edge browser as the note at the bottom right corner suggests.

**YouTube demo:** https://www.youtube.com/watch?v=S738_Z427Ns

This video demos some of the capabilities of Photo editing in Edge browser.

These features might not be available to everyone just yet, but if you wish to try them out early, you can use my GitHub repository to easily use this feature and any other hidden experimental feature in Edge canary.

**Link to the GitHub release related to this feature:**
https://github.com/HotCakeX/MSEdgeFeatures/releases/tag/121.0.2220.0

**Main GitHub repository:**
 https://github.com/HotCakeX/MSEdgeFeatures

**Comment by pipashu** [comment]
How should macOS use it?
